src/components/admin/donations/grid/RenderEditBillingEmailCell.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { AxiosError, AxiosResponse } from 'axios'
+import { useMutation } from '@tanstack/react-query'
+import { useTranslation } from 'next-i18next'
+import { GridRenderEditCellParams } from '@mui/x-data-grid'
+import { TextField, Tooltip, Box } from '@mui/material'
+import Autocomplete from '@mui/material/Autocomplete'
+import { Save } from '@mui/icons-material'
+import { PersonResponse } from 'gql/person'
+import { DonationResponse, UserDonationInput } from 'gql/donations'
+import { useEditDonation } from 'service/donation'
+import { ApiErrors } from 'service/apiErrors'
+import { AlertStore } from 'stores/AlertStore'
+
+interface RenderEditCellProps {
+  params: GridRenderEditCellParams
+  personList?: PersonResponse[]
+  onUpdate(): void
+}
+const addIconStyles = {
+  background: '#4ac3ff',
+  borderRadius: '50%',
+  cursor: 'pointer',
+  padding: 0.7,
+  boxShadow: 3,
+}
+
+export default function RenderEditPersonCell({
```

Comment:
[naming] this should be RenderEditBillingEmailCell right?

Reply:
Yup.